mobile/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -185,6 +185,7 @@
 				9740EEB61CF901F6004384FC /* Run Script */,
 				97C146EA1CF9000F007C117D /* Sources */,
 				97C146EB1CF9000F007C117D /* Frameworks */,
+				F75BBD0B2AC15BE4009A2B6E /* Copy GoogleService-Info.plist */,
```

Comment:
🤔 This looks like a generated name when copying a file in the same directory. Is this correct? I guess it doesn't matter tough as it is only a comment.

Reply:
Yes, this is autogenerated. It's the script which is now being executed to copy over the googleService-info file